src/inc/fixuppointer.h

```diff
@@ -94,6 +122,7 @@ class RelativePointer
         return dac_cast<DPTR(RelativePointer<PTR_TYPE>)>(base)->GetValueMaybeNull(base);
     }
 
+#ifndef DACCESS_COMPILE
```

Comment:
What's the reason for making all these ifdefed out for `DACCESS_COMPILE`? It forces you to write even the code that does not need to be DACized with the DAC cruft. I do not think it is a good idea. Only a fraction of the runtime code has to be DACized.